HW_7/functional.py

```diff
@@ -0,0 +1,67 @@
+# Functional programming
+
+
+def sequential_map(*args):
+    *funcs, elements = args
+    mapped_elements = list()
+    for element in elements:
+        res = element
+        for func in funcs:
+            res = func(res)
+        mapped_elements.append(res)
+    return mapped_elements
+
+
+
+def consensus_filter(*args):
+    *predicates, elements = args
+    filtered_elements = list()
+    for element in elements:
+        satisfies = True
+        for predicate in predicates:
+            if not predicate(element):
+                satisfies = False
+                break
+        if satisfies:
+            filtered_elements.append(element)
+    return filtered_elements

```

Comment:
Опять же переусложнено, применяем `filter` по аналогии с первым заданием
```suggestion
    for predicate in predicates:
        elements = filter(func, elements)
    return list(elements)
```